app/controllers/responses_controller.rb

```diff
@@ -65,10 +65,11 @@ def index
           end
         end
 
-        # get the response, for export, but not paginated
-        @responses = @responses.with_associations.order(:created_at)
+        # Get the response, for export, but not paginated.
+        # We deliberately don't eager load as that is handled in the Results::Csv::Generator class.
+        @responses = @responses.order(:created_at)
 
-        @csv = ResponseCSV.new(@responses)
+        @csv = Results::Csv::Generator.new(@responses)
```

Comment:
FYI Results is the namespace that I'm envisioning containing stuff like Response, Answer, Choice, etc. someday.